app/views/admin/institutions/edit_targets.html.erb

```diff
@@ -0,0 +1,75 @@
+<% content_for(:nav) { render "admin/nav" } %>
+
+<div id="edit-petition-targets">
+  <h2>Petition Targets</h2>
+
+  <%= form_for @actionPage, url: update_targets_admin_action_page_institutions_path do |f| %>
+    <div id="edit-institutions" class="<%= "hidden" unless @action_institutions.present? %>">
+      <%= button_tag "View current targets", type: "button", class: "btn btn-default", id: "view-current-institutions" %>
+      <%= button_tag "Reset targets", type: "button", class: "btn btn-default", id: "reset-institutions" %>
+      <div id="current-institutions" class="hidden">
+        <ul>
+          <% @action_institutions.each do |i| %>
+            <li><%= i.name %></li>
+          <% end %>
+        </ul>
+      </div>
+      <div class="form-item">
+        <%= label_tag :add_institutions, "Add more targets" %>
+        <%= collection_select :institutions_diff, :add_ids, @not_included_institutions, :id, :name,
+          {}, multiple: true, "data-placeholder" => "Start typing to search" %>
+      </div>
+      <div class="form-item">
+        <%= label_tag :remove_institutions, "Remove targets" %>
+        <%= collection_select :institutions_diff, :remove_ids, @action_institutions, :id, :name,
+          {}, multiple: true, "data-placeholder" => "Start typing to search" %>
+      </div>
+    </div>
+
+    <div id="create-action-institutions" class="<%= "hidden" if @action_institutions.present? %>">
+      <div class="form-item">
+        <%= label :institution_category, "Select target type to add in bulk" %>
+        <%= select :institution, :category,
+          options_for_select(Institution.categories.keys.map { |k| [k.humanize, k] }),
+          include_blank: "None" %>
+      </div>
+
+      <% @grouped_institutions.each do |category, institutions| %>
+        <div id="<%= category %>-institutions" class="institutions-by-category hidden">
+          <div class="alert">All <%= category.humanize.pluralize.downcase %> have been added!</div>
+          <%= button_tag "Edit included #{category.humanize.pluralize.downcase}", type: "button",
+            class: "btn btn-default category-show", id: "show-all-#{category}" %>
+          <%= button_tag "Reset targets", type: "button", class: "btn btn-default", id: "reset-institutions" %>
+          <div class="institutions-list">
+            <%= f.collection_check_boxes :institution_ids, institutions, :id, :name,
```

Comment:
It might be nice to instead have a select2 box that allows individual institutions to be removed, like the way editing works.